lib/active_force/active_query.rb

```diff
@@ -33,12 +33,6 @@ def limit limit
       limit == 1 ? to_a.first : self
     end
 
-    def where args=nil, *rest
```

Comment:
`ActiveQuery` is a `Query` with ActiveRecord-like functionality. These methods should remain in `ActiveQuery`.